tests/snuba/api/endpoints/test_organization_stats_v2.py

```diff
@@ -344,7 +361,7 @@ def test_user_org_total_all_accessible(self) -> None:
                 isoformat_z(floor_to_utc_day(self._now)),
             ],
             "groups": [
-                {"by": {}, "series": {"sum(quantity)": [0, 7]}, "totals": {"sum(quantity)": 7}}
+                {"by": {}, "series": {"sum(quantity)": [0, 9]}, "totals": {"sum(quantity)": 9}}
```

Comment:
this is because of the new added outcome https://github.com/getsentry/sentry/blob/0690954123332d22dcebfd47e96945f87fb90be5/tests/snuba/api/endpoints/test_organization_stats_v2.py#L93